src/main/java/seedu/address/logic/commands/ExportTeamCommand.java

```diff
@@ -0,0 +1,135 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import java.awt.Toolkit;
+import java.awt.datatransfer.Clipboard;
+import java.awt.datatransfer.StringSelection;
+import java.util.List;
+
+import javafx.collections.ObservableList;
+import seedu.address.commons.core.index.Index;
+import seedu.address.commons.util.ToStringBuilder;
+import seedu.address.logic.Messages;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.logic.parser.TeamCommandsParser;
+import seedu.address.model.Model;
+import seedu.address.model.contact.Contact;
+import seedu.address.model.team.Team;
+
+/**
+ * Export team members' details to the user's clipboard.
+ */
+public class ExportTeamCommand extends Command {
+
+    public static final String COMMAND_WORD = "export-team";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Copies your team members' details to your clipboard. "
```

Comment:
Can you add the `team` command word here? The format for team commands is slightly different, see `DeleteTeamCommand` and check output of running the `team` command for reference.